arch/arm/dts/sc5xx.dtsi

```diff
@@ -154,17 +154,6 @@
 			clock-names = "spi";
 			status = "okay";
 			bootph-pre-ram;
-
-			flash1: is25lp512@1 {
-				#address-cells = <1>;
-				#size-cells = <1>;
-				compatible = "jedec,spi-nor", "is25lp512";
-				reg = <1>;
-				spi-tx-bus-width = <4>;
-				spi-rx-bus-width = <4>;
-				spi-max-frequency = <10000000>;
-				bootph-pre-ram;
-			};
```

Comment:
When cherry picking use `git cherry-pick -xs`, which adds the original commit hash.

In the commit message also note that this patch was partially applied in the series, hence why we need to add it here.